### ISSUE TITLE
High Humidity Control = Yes in Controller:OutdoorAir

### ISSUE THREAD

**Issue by Alex-Blue** [NREL/EnergyPlus#5406]
When the "high humidity control" = yes in the Controller:OutdoorAir object, it causes the file to crash.

[5ZoneCAVtoVAVWarmestTempFlowErrors.txt](https://github.com/NREL/EnergyPlus/files/83118/5ZoneCAVtoVAVWarmestTempFlowErrors.txt)
[5ZoneCAVtoVAVWarmestTempFlow.txt](https://github.com/NREL/EnergyPlus/files/83119/5ZoneCAVtoVAVWarmestTempFlow.txt)

**Comment by rraustad** [comment]
Know of anyone working on this?

**Comment by mjwitte** [comment]
None that I know of.

**Comment by rraustad** [comment]
Entering Yes for the High Humidity Control field when the field Control High Indoor Humidity Based on Outdoor Humidity Ratio is not included causes a crash.

```
Controller:OutdoorAir,
  Yes,                     !- High Humidity Control
  SPACE1-1,                !- Humidistat Control Zone Name
  1;                       !- High Humidity Outdoor Air Flow Ratio
```

Workaround, enter a Yes or No for Control High Indoor Humidity Based on Outdoor Humidity Ratio

```
Controller:OutdoorAir,
  Yes,                     !- High Humidity Control
  SPACE1-1,                !- Humidistat Control Zone Name
  1,                       !- High Humidity Outdoor Air Flow Ratio
  Yes;                  !- Control High Indoor Humidity Based on Outdoor Humidity Ratio
```

**Comment by rraustad** [comment]
Initial changes complete. See #5552

**Comment by Myoldmopar** [comment]
Closed via #5552
